test/server/cards/03_TWI/units/EncouragingLeadership.spec.ts

```diff
@@ -0,0 +1,40 @@
+describe('EncouragingLeadership', function () {
+    integration(function (contextRef) {
+        describe('EncouragingLeadership\'s ability', function () {
+            it('Gives +1/+1 for this phase to allies', function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['encouraging-leadership'],
```

Comment:
Can we add a unit in hand and play it after Encouraging Leadership to confirm that it isn't affected by the buff... if you find that it is affected, please let me know, there might be an engine bug here I need to fix